proto/Makefile

```diff
@@ -16,35 +16,22 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTO_LIB   := lib/libroachproto.a
-PROTOS      := api.proto config.proto data.proto errors.proto gossip.proto heartbeat.proto internal.proto
+PROTOC      := $(shell git rev-parse --show-cdup)_vendor/usr/bin/protoc
```

Comment:
Why is `git rev-parse` needed here? `make` always cd's into the directory containing the `Makefile` so you should be able to hard-code `../_vendor/...`.